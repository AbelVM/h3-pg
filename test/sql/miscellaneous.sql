SELECT h3_rads_to_degs(h3_degs_to_rads(90.45));

SELECT h3_hex_area_km2(10);
SELECT h3_hex_area_m2(10);

SELECT h3_edge_length_km(10);
SELECT h3_edge_length_m(10);

SELECT h3_num_hexagons(0);
SELECT h3_num_hexagons(15);
