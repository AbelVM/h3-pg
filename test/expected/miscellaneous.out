SELECT h3_rads_to_degs(h3_degs_to_rads(90.45));
 h3_rads_to_degs 
-----------------
           90.45
(1 row)

SELECT h3_hex_area_km2(10);
 h3_hex_area_km2 
-----------------
       0.0150475
(1 row)

SELECT h3_hex_area_m2(10);
 h3_hex_area_m2 
----------------
        15047.5
(1 row)

SELECT h3_edge_length_km(10);
 h3_edge_length_km 
-------------------
       0.065907807
(1 row)

SELECT h3_edge_length_m(10);
 h3_edge_length_m 
------------------
      65.90780749
(1 row)

SELECT h3_num_hexagons(0);
 h3_num_hexagons 
-----------------
             122
(1 row)

SELECT h3_num_hexagons(15);
 h3_num_hexagons 
-----------------
 569707381193162
(1 row)

